Libraries/Text/Text/RCTTextView.m

```diff
@@ -88,11 +88,6 @@ - (BOOL)canBecomeKeyView
 	return _selectable;
 }
 
-- (BOOL)enableFocusRing
```

Comment:
so it already does the right thing that if it isn't something that user can interact with like static text or disabled control we won't see the focus ring?

Reply:
Correct.  If canBecomeKeyView is false, then the field doesn't get focus and so no ring gets drawn.